code/jobs/CreateEnvJob.php

```diff
@@ -20,9 +20,12 @@ public function setUp() {
 	 */
 	public function perform() {
 		echo "[-] CreateEnvJob starting" . PHP_EOL;
+
+		$log = new DeploynautLogFile($this->args['logfile']);
```

Comment:
lol, I double checked to see if the job actually got logfile set in the creation of the job and it did. we just didn't use it?